modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -1004,7 +1004,7 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
 
   const createRng = () => doc.createRange();
 
-  const split = <T extends Node>(parentElm: Node, splitElm: T, replacementElm?: T): T | undefined => {
+  const split = (schema: Schema) => <T extends Node>(parentElm: Node, splitElm: T, replacementElm?: T): T | undefined => {
```

Comment:
The schema is in scope? This should be able to use it directly, the same way other code in this file does?

Reply:
you are right, changed